pkg/common/cns-lib/vsphere/virtualcenter.go

```diff
@@ -553,3 +553,42 @@ func GetVirtualCenterInstance(ctx context.Context,
 	}
 	return vCenterInstance, nil
 }
+
+// GetAllVirtualMachines gets the VM Managed Objects with the given properties from the
+// VM object.
+func (vc *VirtualCenter) GetAllVirtualMachines(ctx context.Context,
```

Comment:
I think this function can be moved to `pkg/common/cns-lib/vsphere/virtualmachine.go`

Reply:
I thought this is similar to `GetHostSystems` which is defined in virtualmachine.go, so this is defined in the virutalcenter.go